diff --git a/python_bindings/setup.py b/python_bindings/setup.py
index 764b589..70bc52d 100755
--- a/python_bindings/setup.py
+++ b/python_bindings/setup.py
@@ -122,8 +122,6 @@ class BuildExt(build_ext):
         if flag in os.environ["CFLAGS"]:
           no_arch_flag=False
           break
-    if no_arch_flag:
-        c_opts['unix'].append('-march=native')
     link_opts = {
         'unix': [],
         'msvc': [],
